pkgs/development/python-modules/isort/default.nix

```
@@ -1,22 +1,24 @@
-{ lib, buildPythonPackage, fetchPypi, isPy27, futures, mock, pytest }:
+{ lib, buildPythonPackage, fetchPypi, isPy27, futures, backports_functools_lru_cache, mock, pytest }:
 
-buildPythonPackage rec {
+let
+  skipTests = lib.optional isPy27 "test_standard_library_deprecates_user_issue_778";
+  testOpts = lib.concatStringsSep " " (map (t: "--deselect test_isort.py::${t}") skipTests);
```

```suggestion
  testOpts = lib.concatStringsSep " " (map (t: "--deselect test_isort.py::${t}") skipTests);
```
You could use `concatImapStrings`.